### ISSUE TITLE
PHP Fatal error: Call to undefined method Consolidation\AnnotatedCommand\Parser\CommandInfo::getInjectedClasses()

### ISSUE THREAD

**Issue by rocketeerbkw** [drush-ops/drush#3985]
**Describe the bug**
Cannot run drush commands after upgrading to 8.2.0.

**To Reproduce**
Upgrade to 8.2.0 and ran `drush status` or any command.

**Expected behavior**
Drush commands work

**Actual behavior**
```
drush status
PHP Fatal error:  Uncaught Error: Call to undefined method Consolidation\AnnotatedCommand\Parser\CommandInfo::getInjectedClasses() in /app/vendor/drush/drush/includes/annotationcommand_adapter.inc:525
Stack trace:
#0 /app/vendor/drush/drush/includes/annotationcommand_adapter.inc(298): annotationcommand_adapter_get_commands_for_commandhandler(Object(Drush\Commands\core\SanitizeCommands), '/app/vendor/dru...')
#1 /app/vendor/drush/drush/includes/annotationcommand_adapter.inc(109): annotationcommand_adapter_get_commands(Array)
#2 /app/vendor/drush/drush/includes/command.inc(1128): annotationcommand_adapter_commands()
#3 /app/vendor/drush/drush/commands/core/topic.drush.inc(38): drush_get_commands()
#4 /app/vendor/drush/drush/includes/command.inc(1508): topic_drush_help_alter(Array)
#5 /app/vendor/drush/drush/includes/drush.inc(338): drush_command_invoke_all_ref('drush_help_alte...', Array)
#6 /app/vendor/drush/drush/includes/context.inc(201): drush_get_global_options()
#7 /app/vendor/drush/drush/includes/context.inc(241): drush_e in /app/vendor/drush/drush/includes/annotationcommand_adapter.inc on line 525
Drush command terminated abnormally due to an unrecoverable error.                                                                    [error]
Error: Uncaught Error: Call to undefined method Consolidation\AnnotatedCommand\Parser\CommandInfo::getInjectedClasses() in
/app/vendor/drush/drush/includes/annotationcommand_adapter.inc:525
Stack trace:
#0 /app/vendor/drush/drush/includes/annotationcommand_adapter.inc(298):
annotationcommand_adapter_get_commands_for_commandhandler(Object(Drush\Commands\core\SanitizeCommands), '/app/vendor/dru...')
#1 /app/vendor/drush/drush/includes/annotationcommand_adapter.inc(109): annotationcommand_adapter_get_commands(Array)
#2 /app/vendor/drush/drush/includes/command.inc(1128): annotationcommand_adapter_commands()
#3 /app/vendor/drush/drush/commands/core/topic.drush.inc(38): drush_get_commands()
#4 /app/vendor/drush/drush/includes/command.inc(1508): topic_drush_help_alter(Array)
#5 /app/vendor/drush/drush/includes/drush.inc(338): drush_command_invoke_all_ref('drush_help_alte...', Array)
#6 /app/vendor/drush/drush/includes/context.inc(201): drush_get_global_options()
#7 /app/vendor/drush/drush/includes/context.inc(241): drush_e in /app/vendor/drush/drush/includes/annotationcommand_adapter.inc,
line 525
```

**Workaround**
None

### System Configuration
| Q               | A
| --------------- | ---
| Drush version?  | 8.2.0
| Drupal version? | 8.6.10
| PHP version     | 7.2.4
| OS?             | Linux

**Additional information**
I wanted to test #3777

**Comment by rocketeerbkw** [comment]
ah, a comedy of errors: it looks like the `Consolidation\AnnotatedCommand\Parser\CommandInfo::getInjectedClasses()` method was added in `consolidation/annotated-command` 2.11.0, the current drush requirement is only `^2.9.1`, and I didn't run `--with-dependencies` when I upgraded drush.

A new install of drush won't have this error.

To fix, make sure you `composer update --with-dependencies drush/drush` or after the fact run `composer update --with-dependencies consolidation/annotated-command` to get the latest version.

I will open a PR to bump the composer.json version requirement.

Related: https://github.com/consolidation/annotated-command/pull/179
